.github/workflows/reproducible-assets.yaml

```
@@ -0,0 +1,93 @@
+name: Reproducible Assets
+on:
+  push:
+  workflow_dispatch:
+jobs:
+  native_asset_build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-22.04, ubuntu-20.04, macos-11, macos-12]
+        time: [now, future]
+    steps:
+      - name: Unbork mac
+        run: |
+          if command -v brew ; then
```

Should/does the step fail if brew is not available?

---

It does not and should not.  If brew is not availble we are either on Linux, which generally has up-to-date tools, or we are on a Mac that we cannot even fix.